### DISCUSSION TITLE
Delete inactive branches on the upstream repository

### DISCUSSION THREAD

**Discussion by ljmf00** [dlang/dmd#13622] [General]
We should avoid publishing branches to the upstream repository and leave them stale, indefinitely. About the ones listed below -- is any of these required other than `stable`?

![image](https://user-images.githubusercontent.com/5050669/152914896-867b1f55-5dfa-4a53-bbfa-efa26d50ca54.png)

**Comment by ibuclaw** [comment]
I'd rather not delete other people's work.

**Reply by ljmf00** [reply]
> I'd rather not delete other people's work.

Sure, but that should be in their tree's, not on the upstream.

**Reply by ljmf00** [reply]
Also, `dcache` is synced with master, so no extra diffs there. @RazvanN7 do you know why `dcache` is there?

**Comment by MoonlightSentinel** [comment]
Removed the `revert-...` branch, that was a remant of the recent CI debugging.

**Comment by maxhaton** [comment]
The newCTFE branch may be seeing some use within the next few days
